Description: Update for changes in gsettings keys for gnome-settings-daemon 3.8

 gnome-control-center (1:3.6.3-0ubuntu30~saucy3) saucy; urgency=medium
 .
   * Apply git patches to support changes in g-s-d 3.8
   * debian/patches:
     - git_power_gsd_proxies.patch,
       Use correct interface for dbus proxy.
     - wacom_update_files_from_gsd.patch,
        update wacom plugin files from g-s-d
Author: Tim Lunn <tim@feathertop.org>
Forwarded: no, Ubuntu Specific


--- a/panels/screen/cc-screen-panel.c
+++ b/panels/screen/cc-screen-panel.c
@@ -350,9 +350,7 @@
                       1, &value,
                       -1);
 
-  /* set both battery and ac keys */
-  g_settings_set_int (self->priv->gsd_settings, "sleep-display-ac", value);
-  g_settings_set_int (self->priv->gsd_settings, "sleep-display-battery", value);
+  g_settings_set (self->priv->session_settings, "idle-delay", "u", value);
 
   set_idle_delay_from_dpms (self, value);
 }
@@ -398,7 +396,7 @@
   i = 0;
 
   /* try to make the UI match the AC setting */
-  value = g_settings_get_int (self->priv->gsd_settings, "sleep-display-ac");
+  g_settings_get (self->priv->session_settings, "idle-delay",  "u", &value);
   do
     {
       gtk_tree_model_get (model, &iter,
@@ -509,7 +507,7 @@
   /* bind the auto dim checkbox */
   widget = WID ("screen_auto_reduce_checkbutton");
   g_settings_bind (self->priv->gsd_settings,
-                   "idle-dim-battery",
+                   "idle-dim",
                    widget, "active",
                    G_SETTINGS_BIND_DEFAULT);
 
